pkg/ccl/changefeedccl/kvfeed/physical_kv_feed.go

```diff
@@ -128,7 +139,11 @@ func (p *rangefeed) addEventsToBuffer(ctx context.Context) error {
 				}
 				stop()
 			case *kvpb.RangeFeedCheckpoint:
-				if !t.ResolvedTS.IsEmpty() && t.ResolvedTS.Less(p.cfg.Frontier) {
+				ev := e.RangeFeedEvent
+				if quantize := p.cfg.WithFrontierQuantize; quantize != 0 {
```

Comment:
I don't know all the cdc code off the top of my head but one mistake I made and then spent some time tracking down in PCR/LDR is that we should only round down if doing so results in a ts that is still >= initial scan ts since otherwise we can prevent the initial scan from ever being considered complete.

Reply:
we sync'ed up offline: this is only an issue in PCR because we quantize a second time when our coordinator is combining progress updates from each processor into the merged frontier and those progress updates are used both to make progress during initial scan and during steady state, which is why they need to be careful about moving a ts down in the former case. This diff here is just the RF case, not initial scans, so this doesn't apply.